images/autobumper/autobump.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET_DIR=$(mktemp -d -t ci-XXXXXXXXXX)
+readonly TARGET_DIR
+
+echo "cloning ${origin} to ${TARGET_DIR} ..."
+git clone --depth 1 "${origin}" "${TARGET_DIR}"
+/usr/bin/autobumper --github-login="${github_login}" --github-token="${github_token}" --git-name="${git_name}" --git-email="${git_email}" --target-dir="${TARGET_DIR}"
```

Comment:
I think I might just prefer to use the raw autobumper command in the image, like we do for `commenter` etc in the infra periodics file

Reply:
Done.